### ISSUE TITLE
java.lang.ClassFormatError javax/ejb/ConcurrencyManagementType with 0.16

### ISSUE THREAD

**Issue by Napfton** [sdaschner/jaxrs-analyzer-maven-plugin#46]
Hi,

just tried to run the maven plugin on my Java EE project, always getting this error:

Failed to execute goal com.sebastian-daschner:jaxrs-analyzer-maven-plugin:0.16:analyze-jaxrs (default) on project -web: Execution default of goal com.sebastian-daschner:jaxrs-analyzer-maven-plugin:0.16:analyze-jaxrs failed: An API incompatibility was encountered while executing com.sebastian-daschner:jaxrs-analyzer-maven-plugin:0.16:analyze-jaxrs: java.lang.ClassFormatError: Absent Code attribute in method that is not native or abstract in class file javax/ejb/ConcurrencyManagementType

Any ideas how to solve this?

Thanks in advance!

**Comment by sdaschner** [comment]
Hi,
Sorry for the late response. It looks like you have some code generation / byte code manipulation process in place? Do you use any special tools as part of your build?

**Comment by Napfton** [comment]
Yeah, we have OpenJPA enhancer plugin running. Without it, it worked. Will it be working anyday in the future with these kind of plugins?

**Comment by sdaschner** [comment]
Hard to say TBH... It depends what the enhancer does with the Bytecode.

How does your build / plugin configuration look like? Maybe as a workaround, Maven profiles would help...?

**Comment by Napfton** [comment]
Profiles are not an option. I Used an older version btw of your plugin, which worked even with the enhancer. The funny thing is: OpenJPA enhance maven plugin only touches Entities, but they are not part of the REST API (that rest API is completely detached from our database)
